File: app/src/main/java/com/tbm/dogs/activities/NotificationDetails.java
package com.tbm.dogs.activities;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import com.tbm.dogs.R;

public class NotificationDetails extends AppCompatActivity {

    TextView tTitle,tBody;

    private void init(){
        tTitle = findViewById(R.id.tTitle);
        tBody = findViewById(R.id.tBody);
    }
    private void initNotification(){
        Intent it = getIntent();
        tTitle.setText(it.getStringExtra("title"));
        tBody.setText(it.getStringExtra("body"));
    }
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_notification_details);
        init();
        initNotification();
    }
}

File: app/src/main/java/com/tbm/dogs/activities/NotificationDetails.kt
package com.tbm.dogs.activities

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView
import com.tbm.dogs.R

class NotificationDetails : AppCompatActivity() {

    lateinit var tTitle: TextView
    lateinit var tBody: TextView

    private fun init() {
        tTitle = findViewById(R.id.tTitle)
        tBody = findViewById(R.id.tBody)
    }

    private fun initNotification() {
        val it = intent
        tTitle.text = it.getStringExtra("title")
        tBody.text = it.getStringExtra("body")
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_notification_details)
        init()
        initNotification()
    }
}
